Formula/neovim.rb

```diff
@@ -128,12 +129,7 @@ def install
     mkdir "build" do
       ohai "Building Neovim."
 
-      build_type =
-        if build.with?("release")
-          "Release"
```

Comment:
should probably still support `build.with?("release")` and do what the user said (Release)